sdk/python/tests/conftest.py

```diff
@@ -22,6 +22,9 @@
 import pandas as pd
 import pytest
 from _pytest.nodes import Item
+from sdk.python.tests.integration.feature_repos.repo_configuration import (
```

Comment:
Shouldn't this be `from tests.integration.feature_repos.repo_configuration`?

Reply:
Fixed.